test/test_models.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import unittest
+
+import torch
+from torchaudio.models import wav2letter
+
+
+class ModelTester(unittest.TestCase):
+    def test_wav2letter(self):
```

Comment:
Should we extend these tests further (i.e. specific input / output tests for fixed weights) or is there little value in it?

Reply:
For what I saw in [torchvision](https://github.com/pytorch/vision/blob/master/test/test_models.py) they had a problem with doing extensive test due to slowing Travis.

The way of the test that I added is that the model should give a letter every 20ms, since the sample rate that they use is 16000, 20ms would be 320 points, however, because of the padding at the 10-th layer it would add 1 extra letter as output.

I used padding same with formula padding = ceil(kernel-stride)/2 due to wav2letter++ [issue](https://github.com/facebookresearch/wav2letter/issues/410#issuecomment-535054856)

Reply:
> I used padding same with formula padding = ceil(kernel-stride)/2 due to wav2letter++ [issue](https://github.com/facebookresearch/wav2letter/issues/410#issuecomment-535054856)

I feel like this should be mentioned in the documentation.